tools/settings.py

```diff
@@ -74,7 +77,7 @@
 # User Settings (env vars)
 ##############################################################################
 _ENV_PATHS = ['ARM_PATH', 'GCC_ARM_PATH', 'GCC_CR_PATH', 'IAR_PATH',
-              'ARMC6_PATH']
+              'ARMC6_PATH', 'PRINT_COMPILER_OUTPUT_AS_LINK']
```

Comment:
@theotherjimmy Is this the right location for this new variable to be defined?

Reply:
Yes.

Reply:
`_ENV_PATHS` should be named `_ENV_PARAMETERS` or similar.

Reply:
Neat. We'll save that renaming for a separate PR.

Reply:
I also wondered at typing this specific line of code, if this would be the right location :-)  I also thought about it to rename the variable right away, but come to the conclusion that this should be a separate PR.